src/bttrx_display.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+This file is part of bt-trx
+
+bt-trx is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+bt-trx is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright (C) 2019 Christian Obersteiner (DL1COM), Andreas Müller (DC1MIL),
+              2020 Florian Wolters (DF2ET)
+Contact: bt-trx.com, mail@bt-trx.com
+*/
+
+#ifdef ARDUINO
+
+#include "bttrx_display.h"
+#include "bttrx_logo.h"
+
+#include "SH1106Wire.h"
+#include "OLEDDisplayUi.h"
+
+extern BTTRX_DISPLAY bttrx_display;
```

Comment:
Not necessary

Reply:
Then includes are required. So I guess only line 30 ```extern BTTRX_DISPLAY bttrx_display;``` want ment here.